docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -197,7 +197,8 @@ The data types of Trino and Kudu are mapped as far as possible:
 +-----------------------+-----------------------+-----------------------+
 | ``CHAR``              | -                     | not supported         |
 +-----------------------+-----------------------+-----------------------+
-| ``DATE``              | -                     | not supported [2]_    |
+| ``DATE``              | ``DATE``              | only supported for    |
```

Comment:
Does this break compatibility for Kudu versions < 1.12? It seems the writes for `DATE` will fail for lower versions now? Or does the kudu-client handle backward compatibility?

cc: @Praveen2112 @Crossoverrr

Reply:
According to the release notes for 1.11:
> Since [KUDU-2625](https://issues.apache.org/jira/browse/KUDU-2625) is addressed, tablet servers now reject individual write operations which violate schema constraints in a batch of write operations. In prior versions the behavior was to reject the whole batch of write operations if a violation of the schema constraints is detected even for a single row. It’s recommended to revise applications which relied on the behavior mentioned above upon upgrading to Kudu 1.11.0.
I don't have thorough understanding of how Trino uses the write API and if it does explore the behavior above. Regarding the client libraries:
>The Kudu 1.11 Java client library is API- and ABI-compatible with Kudu 1.10. Applications written against Kudu 1.10 will compile and run against the Kudu 1.11 client library and vice-versa.
Also for 1.12:
>The Kudu 1.12 Java client library is API- and ABI-compatible with Kudu 1.11. Applications written against Kudu 1.11 will compile and run against the Kudu 1.12 client library and vice-versa.
Aside from not being able to use the new features, it appears to be compatible (I didn't test)